.vale/fixtures/OpenShiftAsciiDoc/MergeConflictMarkers/testvalid.adoc

```diff
@@ -0,0 +1,4 @@
+//Module is included in...
```

Comment:
Not sure what to put in this file. Same with .vale/fixtures/OpenShiftAsciiDoc/MergeConflictMarkers/testinvalid.adoc.